apps/common/models.py

```diff
@@ -58,3 +60,44 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+def activity_image_path(instance, filename):
+    return f'activity-log/{type(instance)._meta.model_name}/{uuid.uuid4()}/{uuid.uuid4()}/{filename}'
+
+
+class BaseActivityLogImage(models.Model):
+    image = models.FileField(
```

Comment:
Let's add another enum char field which will mention where this is used.
`type`

- Payment
- Order
- Package